### DISCUSSION TITLE
Disable animation on specific tui component comp or for all components at once

### DISCUSSION THREAD

**Discussion by max-put** [taiga-family/taiga-ui#10064] [Q&A] [answered]
Hi! There is 2 questios
1. How can I disable animation on particular component (for example TuiFilter, when item become selected - selection border is animated)
2. How can I disable animation as a whole for the app

**Comment by waterplea** [comment, marked as answer]
You can disable animation across all app by providing `TUI_ANIMATIONS_SPEED` as `0` or `TUI_REDUCED_MOTION` as `true`. Animations are also automatically disabled when user has reduced animations set as OS preference. 

Targeting a particular element is a bit trickier, because some animations use CSS, some use Angular Animations. To disable CSS animations set `--tui-duration` CSS variable as 0. To disable Angular Animations provide `TUI_ANIMATIONS_SPEED` as `0` for that particular component.
